### PR TITLE
Add demo page for the new `BitChoiceGroup` component (#1838)

### LINKED ISSUE #1838: Add demo page for the `BitChoiceGroup` component
We need to create a demo page for ChoiceGroup component

### PR DESCRIPTION
**Pull request by sajadkardel**
this closes #1838

### CODE REVIEW

**Comment by msynk** [comment]
@Cyrus-Sushiant plz review

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+using System.Threading.Tasks;
+using Bit.Client.Web.BlazorUI.Components.ChoiceGroup;
+using Bit.Client.Web.BlazorUI.Playground.Web.Models;
+using Bit.Client.Web.BlazorUI.Playground.Web.Pages.Components.ComponentDemoBase;
+
+namespace Bit.Client.Web.BlazorUI.Playground.Web.Pages.Components.ChoiceGroup
+{
+    public partial class BitChoiceGroupDemo
+    {
+        private List<BitChoiceGroupOption> Example_1_5_6_Options;
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs:13]: Plz, Use camelCase or PascalCase to declare in your code

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+                new BitChoiceGroupOption()
+                {
+                   Text = "Option D",
+                   Value = "D",
+                   OnChange = () => {OptionOnChange("D"); }
+                }
+            };
+        }
+
+        public class ChoiceGroupValidationModel
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs:151]: Please, move the class to a separate file.

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs [hunk 3]
```diff
@@ -0,0 +1,485 @@
...
+                Name = "Options",
+                Type = "List<BitChoiceGroupOption>",
+                DefaultValue = "new List<BitChoiceGroupOption>()",
+                Description = "List of options, each of which is a selection in the ChoiceGroup."
+            },
+            new ComponentParameter
+            {
+                Name = "Name",
+                Type = "string",
+                DefaultValue = "Guid.NewGuid().ToString()",
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs:174]: I think this style of introducing the default value is not correct. check it by @msynk

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs [hunk 4]
```diff
@@ -0,0 +1,485 @@
...
+        private List<BitChoiceGroupOption> Example3Options;
+
+        private List<BitChoiceGroupOption> Example4Options;
+
+        private List<BitChoiceGroupOption> Example7Options;
+
+        /// <summary>
+        /// BitChoiceGroupDemo class constructure
+        /// </summary>
+        public BitChoiceGroupDemo()
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs:26]: Maybe better to set the variable at the time of definition and outside the class constructor. This is easier for example. Plz check it by @msynk

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs [hunk 5]
```diff
@@ -0,0 +1,485 @@
...
+                   ImageSize = new System.Drawing.Size(32, 32)
+                },
+                new BitChoiceGroupOption()
+                {
+                    Text = "Pie",
+                    Value = "Pie",
+                    ImageSrc= "https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/choicegroup-pie-unselected.png",
+                    SelectedImageSrc = "https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/choicegroup-pie-selected.png",
+                    ImageAlt = "alt for Pie image",
+                    ImageSize = new System.Drawing.Size(32, 32)
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs:95]: Plz, using `System.Drawing` in this file and remove in this line

**Review by Cyrus-Sushiant — CHANGES_REQUESTED** [review]

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj [hunk 6]
```diff
@@ -13,6 +13,7 @@
         <Content Condition=" '$(BlazorMode)' == 'BlazorWebAssembly' " Remove="Pages\_Layout.cshtml" />
         <Content Remove="compilerconfig.json" />
         <None Include="compilerconfig.json" />
+        <None Include="Pages\Components\ChoiceGroup\BitChoiceGroupDemo.razor" />
```
[the remark this answers is not in the document]
Comment (sajadkardel) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj]: This was added to csproj automatically after adding the demo page files and I forgot to check it out and delete it.
I will do this.

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs (hunk 4 above)
Comment (msynk) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs:25]: yeah, only the ones that need to use instance members of the class should be defined in a constructor.

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs (hunk 3 above)
Comment (msynk) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs:168]: I don't understand what is the problem here.

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs (hunk 5 above)
Comment (msynk) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs]: what?

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs (hunk 3 above)
Comment (msynk) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs:168]: i think its better to use a phrase like this: `a Guid`

**Comment by msynk** [comment]
@sajadkardel plz update the PR based on the new comments.
@Cyrus-Sushiant plz follow up on the results of our discussion.

**Review by maryamhdr — APPROVED** [review]

**Comment by msynk** [comment]
@maryamhdr & @Cyrus-Sushiant 
plz re-review new changes

**Review by Cyrus-Sushiant — APPROVED** [review]

**Review by maryamhdr — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**